ush/atmos_extractvars.sh

```diff
@@ -72,10 +72,16 @@ for outtype in "f2d" "f3d"; do
 
     for infile in "${infile1}" "${infile2}"; do
       if [[ -f "${infile}" ]]; then # check if input file exists before extraction
+        new_infile="${outdirpre}/$(basename "${infile}")_ext"
+        cpfs "${infile}" "${new_infile}"
         # shellcheck disable=SC2312
-        ${WGRIB2} "${infile}" | grep -F -f "${varlist}" | ${WGRIB2} -i "${infile}" -append -grib "${outfile}"
-      else
-        echo "WARNING: ${infile} does not exist."
+        ${WGRIB2} "${new_infile}" | grep -F -f "${varlist}" | ${WGRIB2} -i "${new_infile}" -append -grib "${outfile}"
+        elif [[ -f "${infile}" ]]; then
+          echo "WARNING: ${infile} does not exist in ${com_dir}."
```

Comment:
The `infile` existence check has already been performed on line 74. 
```suggestion
      else
        echo "WARNING: ${infile} does not exist in ${com_dir}."
```